public/css/main.css

```diff
@@ -51,6 +51,10 @@ a:focus {
     font-weight: bold;
 }
 
+.layerButton {
+    max-width: 113px;
+}
+
```

Comment:
Depending on the length of the translated text, it will overflow its container.
I think it is better to "merge" the two buttons into a [button dropdown](https://getbootstrap.com/docs/3.3/components/#btn-dropdowns-single)